common/lib/capa/capa/xqueue_interface.py

```diff
@@ -26,7 +26,7 @@ def make_hashkey(seed):
     Generate a string key by hashing
     """
     h = hashlib.md5()
-    h.update(str(seed))
+    h.update(str(seed).encode('utf-8'))
```

Comment:
@jmbowman in python3 getting error. seed is coming as float value.
```
six.binary_type(seed)
*** TypeError: 'float' object is not iterable

(Pdb) seed
0.8191173181432726

Reply:
In that case, could we use `six.text_type` instead of `str`?  I just want to make sure that we're consistent about the data type used under both versions.  And `str.encode()` under Python 2.7 will fail with `UnicodeDecodeError` if `seed` happens to be non-ASCII bytes.

Reply:
@Ayub-Khan ^^